akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/Effect.scala

```diff
@@ -230,6 +233,64 @@ object Effect {
 
   final case class TimerCancelled(key: Any) extends Effect
 
+  /**
+   * The behavior initiated an ask of a targeted recipient.
+   */
+  final class AskInitiated[Req, Res, T](
+      val target: RecipientRef[Req],
+      val responseClass: Class[Res],
+      createMessage: ActorRef[Res] => Req,
```

Comment:
I think this should not be part of the effect, but already be applied at call site (since that is what will happen in actual runtime) and rather just be the created message